easybuild/easyconfigs/l/libxc/libxc-5.0.0-cpeGNU-20.10.eb

```diff
@@ -0,0 +1,36 @@
+# contributed by Simon Pintarelli, Anton Kozhevnikov and Luca Marsella (CSCS)
+easyblock = 'CMakeMake'
+
+name = 'libxc'
+version = '5.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'cpeGNU', 'version': '20.10'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://www.tddft.org/programs/%(name)s/down.php?file=%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [
+    ('bzip2', '1.0.6'),
+    ('CMake', '3.18.4', '', True),
+]
+
+configopts = [
+    " -DENABLE_FORTRAN=ON -DCMAKE_INSTALL_LIBDIR=lib ",
```

Comment:
Are both these lines needed?

Reply:
Yes, this is a multi-iteration build: the first iteration build the static libraries, while the second line (and second iteration) builds the shared ones.